apps/site/package.json

```diff
@@ -85,6 +92,12 @@
     "tsx": "^4.19.3",
     "typescript": "~5.8.2",
     "typescript-eslint": "~8.29.0",
-    "user-agent-data-types": "0.4.2"
+    "user-agent-data-types": "0.4.2",
+    "@opennextjs/cloudflare": "^1.0.0-beta.3",
+    "wrangler": "^4.13.0",
+    "@flarelabs-net/wrangler-build-time-fs-assets-polyfilling": "^0.0.0"
+  },
+  "optionalDependencies": {
+    "@tailwindcss/oxide-linux-x64-gnu": "~4.1.4"
```

Comment:
I'm adding this following https://github.com/tailwindlabs/tailwindcss/issues/15806#issuecomment-2626536346

I'm not really sure what's going on here to be honest 😕

Reply:
removed, and the Vercel preview seems ok now without this! 😄